Src/BootCamp.Chapter/lesson3.cs

```diff
@@ -0,0 +1,57 @@
+﻿// This file is for hunting bugs only.
+// Completing homework 3 before looking at this is HIGHLY recommended.
+// Try to look at the code in GitHub first. Try to find the mistakes that
+// were made without help or tools first.
+// After that try to find every single thing that seems off.
+// Have fun! :)
+using System;
+using BootCamp.Chap;
+
+namespace BootCamp.Chapter
+{
+    internal class lesson3   // class name must begin with a capital character. 
+    {
+        public static int demo(string count)  // name must begin with a capital character.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename method 'demo' to match pascal case naming rules, consider using 'Demo'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4885581)